### DISCUSSION TITLE
Strange broken behavior on 118 debug

### DISCUSSION THREAD

**Discussion by GL513** [termux/termux-packages#11584] [Q&A]
Recently I encountered an issue after updating packages on the 118 debug. The issue was introduced shortly after the new terminal welcome art was availabile.
ls seems to be completely broken, no matter the directory, and when trying to exit, termux freaks out, and doesn't actually exit, coming up with the command prompt again.
After the freakout and the command prompt is available again, you are able to exit normally, like shown in the image attached.
I'm not a linux developer, more of a user, so I'm not sure how to get the most info out of this other than the logs. I will append them soon.

![Screenshot_20220814-075444_Termux.jpg](https://user-images.githubusercontent.com/83085846/184537023-c0ebd296-3586-4e32-ad87-bf5eb9d76cad.jpg)

**Comment by GL513** [comment]
Dang network issues I thought the other didn't post.

**Reply by aicynide** [reply]
Ol
